mitodl/mitxpro
# Changed wagtail URLs to use course/program readable id

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #462 

#### What's this PR do?
Changes wagtail URLs for detail pages to use course/program readable id instead of the Wagtail standard of using an autogenerated slug

#### How should this be manually tested?
Go to the couse/program listing page in Wagtail and click a link for the live page of any of the detail pages. The URL path should feature the readable id value at the end instead of the slug

#### Screenshots (if appropriate)
![ss 2019-06-11 at 14 16 11 ](https://user-images.githubusercontent.com/14932219/59428711-03d9af80-8dac-11e9-8b5f-8b7aae19467b.png)
